### DISCUSSION TITLE
Component inside model is not working

### DISCUSSION THREAD

**Discussion by richa0892** [livewire/livewire#4871] [1. Help]
I have implemented country, state, and city components. When I call the inside model it is not working. If I call in the page then It is working as expected. When I call inside model then ajax does not fire on the change drop-down

**Comment by denizozturk** [comment]
You can check this example, if you have problems I'd be happy to help

https://gist.github.com/denizozturk/85bee153c03ed2c7d37889434838a8db

**Comment by richa0892** [comment]
[city.zip](https://github.com/livewire/livewire/files/8488189/city.zip)

**Reply by richa0892** [reply]
Please check attachment
